.github/ISSUE_TEMPLATE/CRASH_REPORT.yml

```diff
@@ -61,8 +99,8 @@ body:
     attributes:
       label: Crash Report file
       description: >-
-        Upload your crash report file as an attachment to this issue (drag-and-drop) or to a service such as GitHub
-        Gist (paste a link) and replace this section. This information is critical in resolving your issue!
+        Upload your crash report file as an attachment to this issue (drag-and-drop) or to a service such as [mclo.gs](https://mclo.gs/)
```

Comment:
I don't want users uploading their log files to third-party services, because these paste hosting sites are a dime a dozen and constantly go offline. Users should be directly attaching the log file to the GitHub issue.

Can we somehow use a validation to prevent people from using anything other than GitHub links?

Reply:
nope. the only supported key for any type is `required`.